OsmAnd/res/values/strings.xml

```diff
@@ -50,14 +50,14 @@
     • Added screen to manage all history in the app\n\n
     • Map orientation is not reset after restarting the app\n\n
     • Improved SRTM height marker rendering\n\n
-    • Fixed Arabic display on the map \n\n
+    • Fixed Arabic displayed on the map \n\n
```

Comment:
Leave original.

Reply:
What does "Arabic display on the map" mean?